### DISCUSSION TITLE
Unknown Flashrom Vendor 8037 ID 1240 1240

### DISCUSSION THREAD

**Discussion by fiatbravodriver** [sanni/cartreader#91] [Need help]
Hi hope you are well, 

I've found a new flash rom and wanted to know if there is anything that can be done so it can be re-programmed.

Here are all the details I could get.

![Unknown flashrom](https://user-images.githubusercontent.com/43493640/107624096-483ce300-6c52-11eb-9141-777cf8511ebc.jpg)
![IMG_011](https://user-images.githubusercontent.com/43493640/107624107-4a9f3d00-6c52-11eb-85cd-396c5b9f606c.jpg)

**Comment by sanni** [comment]
Hi,

I have stopped adding new N64 repros after 2 of my 5 suddenly died. But you can try adding the flashrom on your own if you want.

According to the datasheet it is very similar to the already supported S29GL256N.
https://www.cypress.com/file/219926/download
https://www.cypress.com/file/219941/download

You can try changing [line 3040 in N64.ino](https://github.com/sanni/cartreader/blob/master/Cart_Reader/N64.ino#L3040) and see if the error message changes to a different error.

`  else if (strcmp(flashid, "227E") == 0) {`

to

 ` else if (strcmp(flashid, "237E") == 0) {`
